clang/include/clang/CIR/MissingFeatures.h

```diff
@@ -60,7 +60,7 @@ struct MissingFeatures {
   static bool tbaa() { return false; }
   static bool tbaaStruct() { return false; }
   static bool tbaaTagForStruct() { return false; }
-  static bool tbaaTagForEnum() { return false; }
+  static bool tbaaTagForEnum() { return true; }
```

Comment:
I'd say just remove it if nothing else left to do! yay for completion

Reply:
removed